tests/spec/core/dfn-spec.js

```diff
@@ -9,9 +9,9 @@ describe("Core — Definitions", () => {
         "<section id='dfn'><dfn>text</dfn><a>text</a></section>",
     };
     const doc = await makeRSDoc(ops);
-    const $sec = $("#dfn", doc);
-    expect($sec.find("dfn").attr("id")).toEqual("dfn-text");
-    expect($sec.find("a").attr("href")).toEqual("#dfn-text");
+    const sec = doc.querySelector("#dfn");
```

Comment:
Suggestion: `/querySelector\("#(.+)"\)/g` -> `getElementById("$1")`

Reply:
I'm not sure why you resolved this. Please resolve a conversation after proper reasoning.

Reply:
Sorry about that. I just thought since it's a suggestion it's fine either way, besides there are a bunch of references of such querySelector throughout the project. Thank you for pointing out though. Resolved now :smile:

Reply:
Woops, are there? I thought I removed them all 😱 Okay, I'll remove them all in a separate PR.